README.md

```diff
@@ -13,15 +13,15 @@ The [`vscode`](https://github.com/microsoft/vscode) repository is where we do de
 ## Contributing
 If you are interested in fixing issues and contributing directly to the code base, please see the document [How to Contribute](https://github.com/Microsoft/vscode/wiki/How-to-Contribute), which covers the following:
 
-* How to build and run from source
-* The development workflow, including debugging and running tests
-* [Coding guidelines](https://github.com/Microsoft/vscode/wiki/Coding-Guidelines)
-* Submitting pull requests
+* [How to build and run from source](https://github.com/Microsoft/vscode/wiki/How-to-Contribute#build-and-run-from-source)
+* [The development workflow, including debugging and running tests](https://github.com/Microsoft/vscode/wiki/How-to-Contribute#development-workflow)
+* [Coding Guidelines](https://github.com/Microsoft/vscode/wiki/Coding-Guidelines)
```

Comment:
"Guidelines" is capitalized differently to the rest of the list, lower case fits better.